docs/roadmap.md

```diff
@@ -605,7 +605,7 @@ fn printIntRef(x: UnsafeIntRef):
 
 var c: Int = 10
 # "reference" operator
-var ref = UnsafeIntRef(__get_lvalue_as_address(c)) # &Int -> Pointer[Int]
```

Comment:
Should this example have initially been 
`var ref = UnsafeIntRef(Pointer(__get_lvalue_as_address(c)))`?

Reply:
Probably, yes.  In general, this example is out-of-date since the use above even uses `__get_address_as_lvalue` which got removed recently.  For the purposes of this PR, perhaps just revert the change to this file, and the @modularml/mojo-standard-library will work to update this roadmap section in a follow-up PR if that works for you.